composer.json

```diff
@@ -16,7 +16,7 @@
         "jms/serializer-bundle": "^3.0",
         "php-task/php-task": "^1.3.3",
         "php-task/task-bundle": "^1.3 || ^2.0",
-        "sulu/sulu": "^2.0.7 || ^2.1@dev",
```

Comment:
we did `^2.1@dev` here that highest will always test against the newest sulu version also the unreleased we should.